### DISCUSSION TITLE
Ndivia Graphics card choice

### DISCUSSION THREAD

**Discussion by NorthernMunkey** [blakeblackshear/frigate#4639] [Ask A Question]
Hi i'm running frigate on a hp thin client t620plus with a coral usb  stick for obj detection
https://www.parkytowers.me.uk/thin/hp/t620/plus.shtml

i want to populate the pcie slot with a graphics card to offset the ffmpeg load off the cpu and onto a gpu
looking at the below link i just need some clarification, looking in the decoding area do i need to find a card that can do MPEG-1 | MPEG-2   or do I need it to do h.264/265, assuming even though my cameras are h.264 the ffmpeg process is taking the rtsp feed as mpeg? which then means i can choose almost any of those cards, (as long as they fit)
https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new

Thanks for the replies to probably a pretty obvious question

**Comment by NickM-27** [comment]
You want a gpu that can support h.264 and h.265 encode & decoding for sure, the question mainly is how much streams can it handle vs. how many streams do you need it to handle

> assuming even though my cameras are h.264 the ffmpeg process is taking the rtsp feed as mpeg?

if your camera feed is h.264 then that is what the rtsp feed will be encoded as and what the gpu will need to decode as.
